### PR TITLE
Add ability to pause and play the stream by pressing the spacebar

### LINKED ISSUE #579: Enable Spacebar key to pause/play video

### PR DESCRIPTION
**Pull request by thilobillerbeck**
Related to / resolves #579

### CODE REVIEW

**Comment by jeyemwey** [comment]
Can you check how your changes impact the bug in #573?

File: webroot/js/app.js [hunk 1]
```diff
@@ -110,6 +112,7 @@ export default class App extends Component {
     if (this.hasTouchScreen) {
       window.addEventListener('orientationchange', this.handleWindowResize);
     }
+    window.addEventListener('keypress', this.handleKeyPressed);
```
Comment (gingervitis) [inline webroot/js/app.js:115]: I would also remove the listener on Unmount.

**Comment by gingervitis** [comment]
This works for me. Just one little comment and I think we can merge!

**Comment by thilobillerbeck** [comment]
@gingervitis I just added the removal of the event-listener to the components unmount method.

**Pull request merged**